index.html

```diff
@@ -66,6 +64,188 @@ <h2>I’am a Software Developer</h2>
+                sign-ups required.has been the industry's standard
+              </p>
+
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <button class="button" type="button">See Project</button>
+            </div>
+
+            <div class="work-card">
+              <h2>Professional Art Printing Data</h2>
+              <p>
+                A daily selectionof privately personalized reads; no account or
+                sign-ups required.has been the industry's standard
+              </p>
+
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <button class="button" type="button">See Project</button>
+            </div>
+
+            <div class="work-card">
+              <h2>Professional Art Printing Data</h2>
+              <p>
+                A daily selectionof privately personalized reads; no account or
+                sign-ups required.has been the industry's standard
+              </p>
+
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <button class="button" type="button">See Project</button>
+            </div>
+
+            <div class="work-card">
+              <h2>Professional Art Printing Data</h2>
+              <p>
+                A daily selectionof privately personalized reads; no account or
+                sign-ups required.has been the industry's standard
+              </p>
+
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <button class="button" type="button">See Project</button>
+            </div>
+
+            <div class="work-card">
+              <h2>Professional Art Printing Data</h2>
+              <p>
+                A daily selectionof privately personalized reads; no account or
+                sign-ups required.has been the industry's standard
+              </p>
+
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+
+              <button class="button" type="button">See Project</button>
+            </div>
+
+            <div class="work-card">
+              <h2>Professional Art Printing Data</h2>
+              <p>
+                A daily selectionof privately personalized reads; no account or
+                sign-ups required.has been the industry's standard
+              </p>
+              <ul>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+              <button class="button" type="button">See Project</button>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] Please the section has the margins (top, bottom, left, right) defined in the design. This is the [Link to Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) 

- [ ] PLEASE BELLOW ARE THE ORIGINAL DESIGN

![gidi2](https://user-images.githubusercontent.com/63972670/159127404-258d730c-b182-4e2d-a3df-dd340cf35366.JPG)

![gidi1](https://user-images.githubusercontent.com/63972670/159127416-8be86992-f973-4ae0-983b-a7c0e26b755a.JPG)

- [ ] BELLOW ARE WHAT YOU DESIGNED

![WhatsApp Image 2022-03-19 at 4 18 05 PM](https://user-images.githubusercontent.com/63972670/159127458-0ad7c714-4bd6-429b-ac15-1876408fac88.jpeg)

![WhatsApp Image 2022-03-19 at 4 18 05 PM (1)](https://user-images.githubusercontent.com/63972670/159127464-93477fd0-9ece-4232-9d85-7d722ce02490.jpeg)

Reply:
- [ ] As asked by the previous reviewer. Kindly implement it. The section doesn't have the margins (top, bottom, left, right) defined as per the design.
- Your design:
<img width="433" alt="Screenshot 2022-03-19 at 11 25 04 PM" src="https://user-images.githubusercontent.com/70269507/159137369-b0f7e400-493c-4ee6-adf9-aaab9a5a9012.png">

- Figma design:
<img width="247" alt="Screenshot 2022-03-19 at 11 25 52 PM" src="https://user-images.githubusercontent.com/70269507/159137382-1346aa01-6d52-4ffe-ac22-87062fc8a006.png">